prow/config/config_test.go

```diff
@@ -351,14 +351,14 @@ plank:
         default_org: "kubernetes"
         default_repo: "kubernetes"
       gcs_credentials_secret: "default-service-account"
-    'org/bad': # bad
+    'org/inherit':
       timeout: 2h
       grace_period: 15s
       utility_images:
       # clonerefs: "clonerefs:default"
-        initupload: "initupload:default"
-        entrypoint: "entrypoint:default"
-        sidecar: "sidecar:default"
+      # initupload: "initupload:default"
```

Comment:
Why do we comment this out rather than removing it?

Reply:
I kept the old practicing. I guess this is a better way to see what is missing.

Reply:
That is confusing,  you could also write it as `utility_images: {}`.

Also, can you please squash your commits?

Reply:
addressed